openshift/post-deploy-tests.template.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: gabi-post-deploy-tests
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: ${SERVICE_ACCOUNT}
+  imagePullSecrets:
+  - name: ${IMAGE_PULL_SECRET}
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: gabi-post-deploy-tests-${IMAGE_TAG}
+  spec:
+    backoffLimit: 5
+    template:
+      spec:
+        restartPolicy: Never
+        serviceAccountName: ${SERVICE_ACCOUNT}
+        containers:
+          - image: ${IMAGE}:${IMAGE_TAG}
+            imagePullPolicy: Always
+            name: gabi-post-deploy-tests
+            command:
+              - /bin/curl
+            args:
+              - -s
+              - http://localhost:8080/healthcheck
```

Comment:
**To the reviewer:**

This simple curl against the health check end-point will be replaced with a new dedicated test once a new end-point will be added to the service.